src/super_gradients/training/models/classification_models/mobilenetv2.py

```diff
@@ -151,17 +148,17 @@ def __init__(
 
     def forward(self, x):
         x = self.features(x)
-        x = x.mean(3).mean(2)
-        x = self.classifier(x)
-        return x
+        if self.backbone_mode:
+            return x
+        else:
+            x = x.mean(3).mean(2)
```

Comment:
Why did we remove the mean ops in backbone mode ? Wont this be problematic when one would manually replace head ?

Reply:
I think that's the pattern we've been using everywhere in classification models: return a feature map in a backbone mode, apply avg pool -> FC otherwise. Why would you want to change a head of a model in a backbone mode?